data/models/wastemeasurement.py

```diff
@@ -39,7 +39,7 @@ class Meta:
         decimal_places=2,
         blank=True,
         null=True,
-        verbose_name="préparation - masse totale",
+        verbose_name="préparation - masse totale (kg)",
```

Comment:
vu que tu changes les` verbose_name`, en profiter pour mettre des majuscules ?

```suggestion
        verbose_name="Préparation - masse totale (kg)",
```

Reply:
https://docs.djangoproject.com/en/5.1/topics/db/models/#verbose-field-names

> The convention is not to capitalize the first letter of the [verbose_name](https://docs.djangoproject.com/en/5.1/ref/models/fields/#django.db.models.Field.verbose_name). Django will automatically capitalize the first letter where it needs to.